Project/src/MakeCall/FunctionalStreamRenderer.js

```diff
@@ -80,29 +80,29 @@ export const FunctionalStreamRenderer = forwardRef(({ remoteParticipant, stream,
     }
     const isReceivingChanged = () => {
         try {
-            if (stream.isAvailable) {
-                const isReceiving = stream.isReceiving;
-                if (!isReceiving && !isLoading) {
-                    setIsLoading(true)
-                } else if (isReceiving && isLoading) {
-                    setIsLoading(false);
-                }
-            }
+            setIsLoading(stream.isAvailable && !stream.isReceiving);
         } catch (e) {
             console.error(e);
         }
     };
 
+    /*
+     * Returns the amount of videos currently rendered excluding the ScreenSharing
+     */
+    const getCurrentNumberOfRenderers = () => {
+        return document.querySelector('.video-grid-row').querySelectorAll('.stream-container:not([id*="ScreenSharing"]) video').length;
```

Comment:
add refs for renderers and use these instead of querying.

Reply:
not applicable anymore and the solution of rendering will be applied in a following PR